src/Common/Crypto/NullEncryptionStrategy.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+namespace OpenEMR\Common\Crypto;
+
+use OpenEMR\Common\Crypto\EncryptionStrategyInterface;
+
+class NullEncryptionStrategy implements EncryptionStrategyInterface
+{
+    private string $nullVersion = "000";
+
+    public function encryptStandard(?string $value, ?string $customPassword = null, string $keySource = 'drive')
+    {
+        return ($value === null) ? null : $this->nullVersion . base64_encode($value);
```

Comment:
I'm assuming you did this so we could keep track of the fact that the field that was encrypted, instead of just doing an identity function here (return the exact same value)?  That way people could migrate the encryption to a different strategy if needed?